x/dsmr/node_test.go

```diff
@@ -38,6 +40,29 @@ var (
 	chainID = ids.Empty
 )
 
+const testingDefaultValidityWindowDuration = time.Duration(5)
+
+type testingChainIndex struct {
```

Comment:
nit: can we call this something like `testValidityWindow` + an interface check at the top of this file?